integr8ly/cluster-service
# feat: go and dependencies bump

# Description
Bump go and dependencies 

# Verification
* Checkout this branch
*  Verify unit tests pass
```
make vendor/fix
make test/unit
```
* Verify cli can be built successfuly
```
make build/cli
